Create Dockerfile and basic instructions

singiamtel opened the pull request.

singiamtel committed:
Create Dockerfile and basic instructions

screenshakes commented on README.md:
```diff
@@ -27,6 +29,17 @@ It is based on [Discord.Net](https://github.com/discord-net/Discord.Net) and [SK
 
+Alternatively you can use Docker with the provided Dockerfile
+
+Steps 1 to 6 are the same as above
```
All those lines don't seem to break when the markdown is rendered

singiamtel replied:
Oh oops

screenshakes commented on README.md:
```diff
@@ -27,6 +29,17 @@ It is based on [Discord.Net](https://github.com/discord-net/Discord.Net) and [SK
+7) Create a folder called `ROMs/` in the project root and place your game ROM in it
+8) Create a folder called `cores/` in the project root and place the libretro core ([download it here](https://buildbot.libretro.com/nightly/)) in it too
+9) Create a text file called `token` in the project root and paste your bot token in it
+10) ``docker build -e ROM='your_rom_name' -e CORE='your_core_name' -t dew .``. The core name should be specified without the ``_libretro.so`` extension, and the ROM name should be specified without the extension
```
Could you change it to something like "run 'docker ...' in your terminal"?
It's obvious but better be clear for everyone

screenshakes commented on README.md:
```diff
@@ -27,6 +29,17 @@ It is based on [Discord.Net](https://github.com/discord-net/Discord.Net) and [SK
+8) Create a folder called `cores/` in the project root and place the libretro core ([download it here](https://buildbot.libretro.com/nightly/)) in it too
+9) Create a text file called `token` in the project root and paste your bot token in it
+10) ``docker build -e ROM='your_rom_name' -e CORE='your_core_name' -t dew .``. The core name should be specified without the ``_libretro.so`` extension, and the ROM name should be specified without the extension
+11) ``docker run dew`` should now start the bot. If you want to run the bot in the background, use ``docker run -d dew``
```
Same

screenshakes commented:
Hey, thanks a lot for this PR!
I've only added a few comments to the readme changes

singiamtel committed:
Fix markdown render and clarify docker instructions

singiamtel committed:
Include recommended restart policy for docker

The process can ocasionally crash without losing the state of the game, so it's just more convenient to have it relaunch again
